src/simpleResolver.js

```diff
@@ -115,6 +115,12 @@ function compareSemver(a,b) {
   // If either a or b do not have a version element, set a default.
   const aVersion = a.version ?? '0.0.0';
   const bVersion = b.version ?? '0.0.0';
+  if (aVersion.split('.').length < 3) {
+    aVersion = coerce(aVersion)?.raw || '0.0.0';
```

Comment:
Where does `coerce()` get defined?

Reply:
It should be imported from the "semver" library, my bad!

As a side note for why I added it in the first place: the FHIR spec doesn't enforce semantic versioning, and I've encountered runtime errors due to resources with non-compliant versions like "1.0"